main.js

```diff
@@ -173,6 +173,8 @@ async function main () {
   room.on('message', (message) => {
     let msg;
 
+    let from = message.from.substring(0, 9)
```

Comment:
can message be `null` or `undefined` in this case? Probs not a big deal now and a good opportunity for enhancement later is to set a default here or use some typings.

or something like `message && message.from ? message.from.substring(0,9) : ''`

Reply:
It’s an event that an eventEmitter sends, so likely no, but I like defensive programming... nice.

Reply:
For now I will assume the event emitter does to fire without a message.